### ISSUE TITLE
Graphical glitches in 404 page

### ISSUE THREAD

**Issue by libremente** [italia/developers.italia.it#720]
Right now the 404 page has some issues.
1. The `API` link in the Linklist section is set as active
2. Links like "Come lo uso, Come partecipo..." are stiled like "Platform, Software, API", so there is a white line below the section.

![Schermata da 2020-09-29 17-11-34](https://user-images.githubusercontent.com/25111463/94577799-419b6980-0277-11eb-9b7f-6c899b4b9f05.png)

**Comment by shamikakumar** [comment]
Can I work on this issue ?

**Comment by ameybhavsar24** [comment]
@libremente I'd like to work on this issue.
Could you explain some code in [_includes/linklist.html](https://github.com/italia/developers.italia.it/blob/53bd31c0667fa0efc6ca2c6fdbc3a672767b851e/_includes/linklist.html)?
Specifically,
```
    {% if pathArray.size > 0 %}
    {% capture path_1 %}{{ pathArray[1] | remove: '.html' }}{% endcapture %}
    {% capture path_2 %}{{ pathArray[2] | remove: '.html' }}{% endcapture %}
    {% if item.active_by_firstlevel contains path_1 or item.active_by_firstlevel contains path_2 %}
    {% assign isActive = true %}
    {% endif %}
    {% endif %}
```
Some examples when isActive is set to true will be enough.

**Comment by bfabio** [comment]
@ameybhavsar24 Sorry I'm on my phone right now, but you can print variable right on the page with `{{  path_1 }}`, `{{ item.active_by_firstlevel }}`, etc. to show their contents.

**Comment by bfabio** [comment]
@ameybhavsar24 Whoops, sorry. The fix for the English homepage in #751 resolved this as well.
